### PR TITLE
[Use-case] Add example use-case

### PR DESCRIPTION
**Pull request by supercrazysam**
This is a follow-up from https://discuss.ardupilot.org/t/cmu-nrec-falco-drone-safe-flights-by-agile-collision-avoidance/83300

Adding "Safe flights by agile collision avoidance" to example use-case.

Sincerely,
Sam Shum

### CODE REVIEW

**Comment by Hwurzburg** [comment]
did you mean to include the CSS file? probably need to rebase on master/delete it....also squash the commits to one...

**Comment by supercrazysam** [comment]
> did you mean to include the CSS file? probably need to rebase on master/delete it....also squash the commits to one...

Oh no, for the whole commit, it only consists of text update on the "use-case" page, and also one image upload. That's it.

**Comment by supercrazysam** [comment]
> did you mean to include the CSS file? probably need to rebase on master/delete it....also squash the commits to one...

Squashed everything and removed the CSS. The extra CSS was generated by "python update.py --site copter"

File: common/source/docs/common-use-cases-and-applications.rst [hunk 1]
```diff
@@ -21,6 +21,17 @@ Use Cases
 
 .. image:: ../../../images/case-tractor.jpg
 
+
+- `Safe flights by agile collision avoidance <https://discuss.ardupilot.org/t/cmu-nrec-falco-drone-safe-flights-by-agile-collision-avoidance/83300>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-use-cases-and-applications.rst:25]: After looking at the built page, this tilte needs working a bit....am going to merge it and then tweak it....thanks

**Comment by Hwurzburg** [comment]
thanks...merging

File: common/source/docs/common-use-cases-and-applications.rst (hunk 1 above)
Comment (supercrazysam) [reply common/source/docs/common-use-cases-and-applications.rst:25]: Hi @Hwurzburg, after talking to the team, I think we can change this the title to "Autonomous drone with agile collision avoidance"

**Comment by supercrazysam** [comment]
> thanks...merging

Hi @Hwurzburg, after talking to the team.  

If it is more suitable, I think we can change this the title to "Autonomous drone with agile collision avoidance".

**Comment by Hwurzburg** [comment]
Already changed....needs to be shorter but descriptive...I changed it to Collision Avoidance...

**Comment by supercrazysam** [comment]
> Already changed....needs to be shorter but descriptive...I changed it to Collision Avoidance...

Hi @Hwurzburg, thank you for letting me know.

I understand. But would it be possible to change it to "autonomous drone with collision avoidance" ?
It is the request from our team. Thank you!

**Pull request merged**